duffy/models/baremetal_nodes.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+import datetime
+import uuid
+
+from duffy.database import db, Duffyv1Model
+from duffy.extensions import marshmallow
```

Comment:
We shouldn't need the marshmallow import here if we split these classes into new files

Reply:
yes, will fix it